### PR TITLE
1.2.6 Tidb-operator-lightning

### PR DESCRIPTION
**Pull request by shonge**

### CODE REVIEW

File: session2/chapter1/tidb-operator-lightning.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+![整体架构](https://download.pingcap.com/images/docs-cn/v3.1/tidb-lightning-architecture.png)
+
+TiDB Lightning 主要包含两个部分：
+
+- **`tidb-lightning`**（“前端”）：主要完成适配工作，通过读取数据源，在下游 TiDB 集群建表、将数据转换成键值对（KV 对）发送到 `tikv-importer`、检查数据完整性等。
+- **`tikv-importer`**（“后端”）：主要完成将数据导入 TiKV 集群的工作，对 `tidb-lightning` 写入的键值对进行缓存、排序、切分操作并导入到 TiKV 集群。
+
+## 操作
+
+### 1. 通过Mydumper执行全量逻辑备份
```
Comment (cofyc) [inline session2/chapter1/tidb-operator-lightning.md]:
```suggestion
### 1. 通过 Mydumper 执行全量逻辑备份
```

File: session2/chapter1/tidb-operator-lightning.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+mysql.db-schema.sql                       mysql.tables_priv-schema.sql
+mysql.default_roles-schema.sql            mysql.tidb-schema.sql
+mysql.expr_pushdown_blacklist-schema.sql  mysql.tidb.sql
+mysql.gc_delete_range-schema.sql          mysql.user-schema.sql
+mysql.gc_delete_range_done-schema.sql     mysql.user.sql
+mysql.global_priv-schema.sql              test-schema-create.sql
+mysql.help_topic-schema.sql
+```
+
+### 2. 使用Lightning恢复数据
```
Comment (cofyc) [inline session2/chapter1/tidb-operator-lightning.md]:
```suggestion
### 2. 使用 Lightning 恢复数据
```

File: session2/chapter1/tidb-operator-lightning.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
+# 在 Kubernetes 集群上使用 Lightning 导入数据
+
+## 背景
+
+Mydumper + Loader 使用多线程导入导出数据，但在导入数据时需要经过 TiDB SQL 语法解析，导致 TiDB 计算能力成为新的瓶颈。所以又一个想法孕育而出——导入数据不经过 SQL 解析，直接转换成KV键值对写入 TiKV 集群。
```
Comment (cofyc) [inline session2/chapter1/tidb-operator-lightning.md]:
```suggestion
Mydumper + Loader 使用多线程导入导出数据时需要经过 TiDB SQL 语法解析，导致 TiDB 计算能力成为新的瓶颈。所以又一个想法孕育而出——导入数据不经过 SQL 解析，直接转换成 KV 键值对写入 TiKV 集群。
```

File: session2/chapter1/tidb-operator-lightning.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+## 介绍
+
+TiDB Lightning 整体架构
```
Comment (cofyc) [inline session2/chapter1/tidb-operator-lightning.md]:
```suggestion
TiDB Lightning 整体架构：
```

File: session2/chapter1/tidb-operator-lightning.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+- 创建备份所需的 secret
+
+kubectl create secret generic backup-secret --namespace=test-backup --from-literal=user=root --from-literal=password='root_password'
+
+`helm install pingcap/tidb-backup --version=v1.1.0-beta.2 --name backup-cluster-1 --namespace test-backup --set-string clusterName=cluster-1,storage.size=500Gi`
+
+- 确认备份任务完成
+
+kubectl -n test-backup get job -l app.kubernetes.io/instance=backup-cluster-1
```
Comment (cofyc) [inline session2/chapter1/tidb-operator-lightning.md:52]: put shell comments in code block labeled with `shell` or `bash`.

File: session2/chapter1/tidb-operator-lightning.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+
+```
+NAME                            COMPLETIONS   DURATION   AGE
+basic-fullbackup-202003080800   1/1           3s         3m32s
+
```
Comment (cofyc) [inline session2/chapter1/tidb-operator-lightning.md]:
```suggestion
```

File: session2/chapter1/tidb-operator-lightning.md [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+- 检查恢复任务状态
+
+kubectl -n test-cluster get job -l app.kubernetes.io/name=restore-cluster-1-tidb-lightning
+
+```
+NAME                               COMPLETIONS   DURATION   AGE
+restore-cluster-1-tidb-lightning   1/1           3s         9m3s
+```
+
+- 确认数据恢复情况
```
Comment (DanielZhangQD) [inline session2/chapter1/tidb-operator-lightning.md]:
```suggestion
- 访问 `cluster-2` TiDB 服务，确认数据恢复情况
```

File: session2/chapter1/tidb-operator-lightning.md (hunk 5 above)
Comment (shonge) [reply session2/chapter1/tidb-operator-lightning.md:52]: done

**Review by tennix — APPROVED** [review]
LGTM

**Pull request merged**